tox.ini

```diff
@@ -52,14 +52,14 @@ deps=
     bandit
 commands=
     # Skip B101 low severity assert warnings in the tests directory
-    bandit -r ./tests --severity-level low  --exit-zero --skip B101
+    bandit -r ./tests --severity-level low --exit-zero --skip B101,B113
```

Comment:
I'm not sure why we're adding things to `--skip` here.  It should not affect the correctness of the CI.  Why'd you want to update this?

The original purpose of the two separate testenvs was:

- `bandit` configures the tool to detect only those issues we want CI to enforce
- `bandit-exitzero` configures the tool to find everything it can, but does not enforce the results it finds

It's a valid question whether this is actually working out properly and I wouldn't be surprised if we end up scrapping the exitzero env at some point.  But as long as it exists, I don't get why we'd be adding things into `skip` here as it seems to defeat the original intended purpose.

Reply:
I misunderstood the purpose of the bandit-exitzero testenv and did this to avoid warning fatigue. (I assumed this could be added, since B113 in tests is almost guaranteed to be a false positive.) Changed.